contracts/SafeProtocolRegistry.sol

```diff
@@ -44,21 +46,24 @@ contract SafeProtocolRegistry is ISafeProtocolRegistry, Ownable2Step {
      * @param module Address of the module
      * @param moduleType Enum.ModuleType indicating the type of module
      */
-    function addModule(address module, Enum.ModuleType moduleType) external virtual onlyOwner {
+    function addModule(address module, uint8 moduleType) external virtual onlyOwner {
+        _addModule(module, moduleType);
```

Comment:
Why is the internal function needed? Also, can it be modified to be able to pass multiple types?

Reply:
Done, added test as well